tests/test_app_upload_jurisdictions_file.py

```diff
@@ -0,0 +1,179 @@
+import os, math, uuid
+import tempfile
+import json, csv, io
+
+import pytest
+
+from models import Election, JurisdictionAdministration, User
+from arlo_server import app, db
+
+
+def post_json(client, url, obj):
+    return client.post(url, headers={'Content-Type': 'application/json'}, data=json.dumps(obj))
+
+
+@pytest.fixture
+def client():
+    app.config['TESTING'] = True
+    client = app.test_client()
+
+    with app.app_context():
+        db.drop_all()
+        db.create_all()
+
+    yield client
+
+    db.session.commit()
```

Comment:
are `post_json` and `client` copy-pasted from `test_app.py`? I imported them in `test_report.py`. Do you think one way or the other is better? Or should we pull them out into a helper module?

Reply:
They were just copied, yeah. We could put them in `tests/utils.py` or something.